Sources/ToosieSlide/UICollectionView+Extensions.swift

```diff
@@ -18,4 +18,31 @@ public extension UICollectionView {
   var carouselFlowLayout: UICollectionViewCarouselLayout {
     collectionViewLayout as! UICollectionViewCarouselLayout
   }
+  
+  /// Scrolls the CollectionView to a given cell index.
+  ///
+  /// If the passed index is grater than the number of items in the collection, the scroll stops at the last item.
+  /// - Parameters:
+  ///   - index: The index of desired cell. Should be a value greater than or equal to 0.
+  ///   - animated: Whether or not to animate the scroll.
+  func scrollToCell(at index: CellIndex, animated: Bool = true) {
+    assert(index >= 0, "`index` cannot be negative.")
+    // check if delegate allows it.
+    guard (delegate as? UICollectionViewDelegateCarouselLayout)?.collectionView(self, shouldDisplayCellAt: index) != false else { return }
+    // make sure index don't overflow
+    let index = min(index, numberOfItems(inSection: 0) - 1)
```

Comment:
if numberOfItems is 0, this min will return -1

Reply:
✅